lib/setupmanagers/qemuhck/qemu_machine.rb

```diff
@@ -684,6 +684,7 @@ def run(scope, run_opts = nil)
 
       process_device_commands
       normalize_lists
+      @configured = @run_opts[:configured] if [true, false].include?(@run_opts[:configured])
```

Comment:
To make this simpler and torelate other truthy and falsy values:

```suggestion
      @configured = @run_opts[:configured] if @run_opts.key?(:configured)
```